diff --git forkSrcPrefix/Makefile forkDstPrefix/Makefile
index f5e3bf2e347c24fdc6bea0b9149ba867ed0c1abf..5a3f2817f7f520e5460aedecdeb90a2e16cda68d 100644
--- forkSrcPrefix/Makefile
+++ forkDstPrefix/Makefile
@@ -536,6 +536,19 @@ else ifeq ($(platform), miyoomini)
 	ARCH = arm
 	BUILTIN_GPU = neon
 
+# TRIMUI MODEL S
+else ifeq ($(platform), trimui)
+  TARGET := $(TARGET_NAME)_libretro.so
+  CC  = $(CROSS_COMPILE)gcc
+  CXX = $(CROSS_COMPILE)g++
+  AR  = $(CROSS_COMPILE)ar
+  fpic := -fPIC -DPIC
+  SHARED := -shared -nostdlib -Wl,-version-script=link.T
+  CFLAGS += -fomit-frame-pointer -ffast-math -march=armv5te -mtune=arm926ej-s
+  CFLAGS += -DSMALL_TRANSLATION_CACHE
+  HAVE_DYNAREC := 1
+  CPU_ARCH := arm
+  
 # Windows
 else
 	TARGET := $(TARGET_NAME)_libretro.dll
