diff --git forkSrcPrefix/Makefile.libretro forkDstPrefix/Makefile.libretro
index 9660e99e3c4a2d22bf2918e4dc2781a481d8b060..3ab75c1de196add572d6e6af8cf1ed6b0c39e255 100644
--- forkSrcPrefix/Makefile.libretro
+++ forkDstPrefix/Makefile.libretro
@@ -484,6 +484,17 @@ else ifeq ($(platform), miyoo)
 	CFLAGS += -ffast-math -fomit-frame-pointer -march=armv5te -mtune=arm926ej-s
 	CFLAGS += -DDINGUX -DRS90
 
+else ifeq ($(platform), trimui)
+	TARGET := $(TARGET_NAME)_libretro.so
+	CC  = $(CROSS_COMPILE)gcc
+	CXX = $(CROSS_COMPILE)g++
+	AR  = $(CROSS_COMPILE)ar
+	SHARED := -shared -nostdlib -Wl,-version-script=libretro/link.T
+	fpic := -fPIC -DPIC
+	CFLAGS += $(PTHREAD_FLAGS)
+	CFLAGS += -ffast-math -fomit-frame-pointer -march=armv5te -mtune=arm926ej-s
+	CFLAGS += -DDINGUX -DRS90
+
 # GCW0
 else ifeq ($(platform), gcw0)
 	TARGET := $(TARGET_NAME)_libretro.so
